webserver/views/api.py

```diff
@@ -248,20 +250,30 @@ def _messybrainz_lookup(listens):
     return augmented_listens
 
 
-def _validate_listen(listen):
+def _validate_listen(listen, listen_type):
```

Comment:
- The set size is just 3 so it would hardly make any difference.
- Updated the code to use the 3 defined variables.